docs/gitrepo-add.md

```diff
@@ -1,17 +1,17 @@
 # Registering
 
 ## Proper namespace
-Git repos are added to the Fleet manager using the `GitRepo` custom resource type. The `GitRepo` type is namespaced. By default, Rancher will create two Fleet workspaces: **fleet-default** and **fleet-local**. 
```

Comment:
It is ok here, but I think it is better to put cosmetic changes like the removal of trailing white-spaces in a separate commit to make the changes easier to verify/read.

Reply:
Yes, good point.